### PR TITLE
NAS-118352 / 22.12 / Fixed train

### PR DESCRIPTION
**Pull request by RehanY147**

### CODE REVIEW

File: src/app/pages/system/update/update.component.ts [hunk 1]
```diff
@@ -114,9 +114,8 @@ export class UpdateComponent implements OnInit {
           this.checkable = true;
           this.fullTrainList = trains.trains;
 
-          // On page load, make sure we are working with train of the current OS
-          this.train = trains.current;
-          this.selectedTrain = trains.current;
+          this.train = trains.selected;
+          this.selectedTrain = trains.selected;
```
Comment (AlexKarpov98) [inline src/app/pages/system/update/update.component.ts:118]: @RehanY147 why do we need 2 same values in 2 different variables?

Comment (RehanY147) [reply src/app/pages/system/update/update.component.ts:118]: One is the value of the `ix-select` and one is to remember what the original selected train was so we don't show a warning when they switch back to that.

**Review by denysbutenko — APPROVED** [review]
image

select border color is wrong otherwise looks good

**Comment by RehanY147** [comment]
NAS-118390 will handle the layout issues

**Pull request merged**